transport/handler/routes.go

```diff
@@ -59,7 +59,10 @@ func (h *Handler) setupRoutes() {
 					r.Patch("/", middleware.RequireAuthUser(h.updateMemberRole))
 				})
 			})
-			r.Get("/repository/tags", middleware.RequireAuthUser(h.listGithubRepositoryTags))
+			r.Route("/github-repository", func(r chi.Router) {
+				r.Post("/", middleware.RequireAuthUser(h.setGithubRepoForProject))
+				r.Get("/tags", middleware.RequireAuthUser(h.listGithubRepositoryTags))
```

Comment:
Just note to myself, rename `listGithubRepositoryTags` -> `listGithubRepoTags`

Reply:
But then also client methods and other places?..
Because indeed, `GithubRepo` / `Repo` sounds better; `Repository` gets confused with repository pattern a bit.

Reply:
(optional, minor) Also, it might be a very, very nice to maybe start creating a dictionary in `README`.
Agreeing on one term and stick to it and define it there. Ubiquitous language - DDD principle.
In our case, `GitTag` for example - might be nice to define that practically, it's always GitHub tag...

Reply:
I like it 👍 

https://github.com/jan-zabloudil/release-manager/issues/122 added